### PR TITLE
changes

### PR DESCRIPTION
**Pull request by ccfredbaum**
Simplified scenario icons and text

### CODE REVIEW

File: my-video-room/module/sitevideo/views/module-admin.php [hunk 1]
```diff
@@ -22,14 +22,9 @@
 	?>
 <div class="mvr-admin-page-wrap">
 	<h2>
-		<?php esc_html_e( 'Site Conference Center Settings', 'my-video-room' ); ?>
+		<?php esc_html_e( 'Site Conference Center Settings', 'my-video-room' ) . Factory::get_instance( ModuleConfig::class )->module_activation_button( MVRSiteVideo::MODULE_SITE_VIDEO_ID );  ?>
```
Comment (alecsammon) [inline my-video-room/module/sitevideo/views/module-admin.php:25]: is there a reason to put the button into the title? I don't think it is part of the actual title. Is this is a styling thing? Can we use `inline-block`, or `float: left` instead?

File: my-video-room/dao/class-moduleconfig.php [hunk 2]
```diff
@@ -359,13 +361,8 @@ public function module_activation_button( int $module_id ): string {
...
 		?>
-		<div style="display: flex; justify-content: space-between; width: 50%;">
-			<a href="<?php echo esc_url( $url ); ?>" class="button button-primary <?php echo esc_attr( $type ); ?>">
-				<?php echo esc_html( $main_text ); ?>
-			</a>
-			<a href="<?php echo esc_url( $sub_tab_url ); ?>" class="button button-primary">
-				<?php echo esc_html( $action_text ); ?>
-			</a>
+		<div>
+			<a href="<?php echo esc_url( $url ); ?>" class="<?php echo esc_html( $status ); ?>" title="<?php echo esc_html( $description ); ?>"><i class="dashicons <?php echo esc_html( $type ) . ' ' . esc_html( $status ); ?>"></i> <?php echo esc_html( $main_text ); ?></a>
```
Comment (alecsammon) [inline my-video-room/dao/class-moduleconfig.php:365]: can we multiline this?
```
<a href="<?php echo esc_url( $url ); ?>" 
    class="<?php echo esc_html( $status ); ?>" 
    title="<?php echo esc_html( $description ); ?>"
>
    <i class="dashicons <?php echo esc_html( $type ) . ' ' . esc_html( $status ); ?>"></i> 
    <?php echo esc_html( $main_text ); ?>
</a>
```

File: my-video-room/css/template.css [hunk 3]
```diff
@@ -31,6 +31,18 @@ ul.myvideoroom-outer-nav-tab-wrapper {
 	margin-bottom: 10px!important;
 }
 
+.mvr-icons-enabled {
```
Comment (alecsammon) [inline my-video-room/css/template.css:34]:
```
.mvr-icons-enabled,
.mvr-icons-disabled {
	font-size: 18px;
	text-decoration: none;
}
```
.mvr-icons-enabled {
    color: green;
}

mvr-icons-disabled {
    color: red;
}
```

File: my-video-room/dao/class-moduleconfig.php (hunk 2 above)
Comment (ccfredbaum) [reply my-video-room/dao/class-moduleconfig.php:365]: done

File: my-video-room/css/template.css (hunk 3 above)
Comment (ccfredbaum) [reply my-video-room/css/template.css:34]: I see how you do it now - thanks

File: my-video-room/dao/class-moduleconfig.php (hunk 2 above)
Comment (ccfredbaum) [reply my-video-room/dao/class-moduleconfig.php:365]: done

File: my-video-room/module/sitevideo/views/module-admin.php (hunk 1 above)
Comment (ccfredbaum) [reply my-video-room/module/sitevideo/views/module-admin.php]: done

**Pull request merged**